.github/workflows/inspektor-gadget.yml

```diff
@@ -117,7 +117,7 @@ jobs:
       with:
         tag_name: ${{ github.ref }}
         release_name: Release ${{ github.ref }}
-        draft: true
+        draft: false
```

Comment:
Since the release will be created automatically, we need some script to write the release description with changelog

Reply:
I agree. What about having a CHANGELOG file updated manually in each import PR and having a script that reads and puts it in the releases notes?

Reply:
/cc @joaquimrocha

Reply:
Since github releases can be edited after being published, I suggest we can merge this PR first, and add the CHANGELOG automation later. Meanwhile, the release description can be edited manually.

Reply:
Nice. Updated the PR to also include windows